docs/setup.md

```diff
@@ -14,23 +14,44 @@ The **Global Visibility** solution pack depends on the following solution packs
 | Solution Pack Name | Version         | Purpose                                       |
 |:-------------------|:----------------|:----------------------------------------------|
 | SOAR Framework     | v2.0.1 or later | Required for Alerts, Assets, Incident modules |
+| SOC Simulator      | v1.0.2 or later | Required for record simulation                |
 
 Before installing the FortiSOAR Global Visibility Solution Pack, ensure the following prerequisites are met:
 
 1. A working FortiSOAR instance at each site
 2. Access to the FortiSOAR API of each instance
-3. User credentials with appropriate permissions to access data
+3. User credentials with appropriate permissions to access data using API
 4. Remote FortiSOAR connector for Global Visibility data aggregation
 
 # Configuration
 
-- The **Global Visibility** solution pack launches the configuration wizard as soon as it finishes installation. However, You can also click the <strong>Configure</strong> button on the solution pack's installation page to configure the sites.
+## Install and Configure Connectors
+Install and configure the following connectors:
 
+- **Remote FortiSOAR** - This connector facilitates automated interactions, with a FortiSOAR endpoint using FortiSOAR™ APIs. To configure and use the Remote FortiSOAR connector, refer to [Configuring Remote FortiSOAR](https://docs.fortinet.com/fortisoar/connectors/remotefsr)
 
-- Remote FortiSOAR connector installs with the solution pack. You need to configure the connector with relevant sites to use the dashboard widgets.
+## Setup Global Visibility on FortiSOAR
+After installation of the **Global Visibility** solution pack, run the configuration wizard to ready your FortiSOAR environments to reterive information from them. This wizard helps you to configure other FortiSOAR sites.
+
+1. Log in to FortiSOAR, after [installation](#installation) completes.
+
+2. Click the button **Configure** from the lower-left of the screen.
+
+    ![Global Visibility start configuration](./res/config-wizard-00.png)
+
+3. Click the button **Let's get started** on the Outbreak Response Framework configuration page.
+
+    ![Global Visibility get started](./res/config-wizard-01.png)
+
+4. Configure Remote FortiSOAR connector. Add multiple connector configurations to retreive information from multiple sites and click **Next**.
```

Comment:
4. Configure Remote FortiSOAR connector. Add multiple connector configurations to retrieve information from multiple sites and click **Next**.

'Retrieve'